ci-operator/step-registry/openshift/microshift/infra/conformance-setup/openshift-microshift-infra-conformance-setup-commands.sh

```diff
@@ -1,5 +1,30 @@
 #!/bin/bash
 set -xeuo pipefail
 
+IP_ADDRESS="$(cat ${SHARED_DIR}/public_address)"
+HOST_USER="$(cat ${SHARED_DIR}/ssh_user)"
+INSTANCE_PREFIX="${HOST_USER}@${IP_ADDRESS}"
+
+echo "Using Host $IP_ADDRESS"
+
+mkdir -p "${HOME}/.ssh"
+cat <<EOF >"${HOME}/.ssh/config"
+Host ${IP_ADDRESS}
+  User ${HOST_USER}
+  IdentityFile ${CLUSTER_PROFILE_DIR}/ssh-privatekey
+  StrictHostKeyChecking accept-new
+  ServerAliveInterval 30
+  ServerAliveCountMax 1200
+EOF
+chmod 0600 "${HOME}/.ssh/config"
+
 cp /go/src/github.com/openshift/microshift/origin/skip.txt "${SHARED_DIR}/conformance-skip.txt"
 cp "${SHARED_DIR}/conformance-skip.txt" "${ARTIFACT_DIR}/conformance-skip.txt"
+
+# Disable workload partitioning for annotated pods to avoid throttling.
+ssh "${INSTANCE_PREFIX}" "sudo sed -i 's/resources/#&/g' /etc/crio/crio.conf.d/11-microshift-ovn.conf"
+ssh "${INSTANCE_PREFIX}" "sudo systemctl daemon-reload"
+# Just for safety, restart everything from scratch.
+ssh "${INSTANCE_PREFIX}" "echo 1 | sudo microshift-cleanup-data --all"
+ssh "${INSTANCE_PREFIX}" "sudo systemctl restart crio"
+ssh "${INSTANCE_PREFIX}" "sudo systemctl start microshift"
```

Comment:
Do we want to enable microshift service so that it's restarted by systemd if it crashes?
Or, we start on purpose not to miss crashes?

If it's the latter, let's add a comment please.

Reply:
It is not enabled on purpose. If it crashes that shouldnt happen and we need to address it, so better to force a failure.